lib/writer.js

```diff
@@ -147,3 +162,44 @@ function currentDate() {
 
   return util.format('%d-%s-%s', now.getFullYear(), pad(now.getMonth() + 1), pad(now.getDate()));
 }
+
+function nonGithubUrl(url) {
+  try {
+    var idx = url.indexOf("@");
+    if (idx !== -1) {
+      url = url.slice(idx + 1).replace(/:([^\d]+)/, "/$1");
+    }
+    url = url_.parse(url);
+    var protocol = url.protocol === "https:"
+      ? "https:"
+      : "http:"
+    return protocol + "//" + (url.host || "") +
+      url.path.replace(/\.git$/, "");
+  }
+  catch(e) {}
+}
+
+function getPackageData(pkg) {
+  var pkgData = JSON.parse(fs.readFileSync(pkg, {encoding:'utf8'})) || {};
```

Comment:
We can't use require() because it will require relative to the conventional-changleog library, not the cwd.